src/Makefile.qt.include

```diff
@@ -193,6 +193,8 @@ BITCOIN_QT_H = \
   qt/winshutdownmonitor.h
 
 RES_ICONS = \
+  qt/res/icons/about.png \
```

Comment:
Are these two images used?  I never found a reference.

Reply:
it use in bitcoinui.cpp. i added it back